ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/eth/eth_getProof_blockHash.json

```diff
@@ -20,7 +20,7 @@
       "nonce" : "0x0",
       "storageHash" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
       "storageProof" : [ {
-        "key" : "0x347",
+        "key" : "0x0000000000000000000000000000000000000000000000000000000000000347",
```

Comment:
I don't think this will match geth either? the hive test example had 0x01 - so just enough zeros to make it an even number of chars

Reply:
my mistake - in the example 0x01 is passed in as a parameter so that's what comes out.

Reply:
hmmm I ran geth and besu and put results in the ticket - it looks like we're both trimming zeros so either I read the hive tests wrong or idk